glide.yaml

```diff
@@ -114,7 +114,7 @@ import:
 - package: github.com/mattn/go-sqlite3
   version: 3fb7a0e792edd47bf0cf1e919dfc14e2be412e15
 - package: github.com/lib/pq
-  version: 50761b0867bd1d9d069276790bcd4a3bccf2324a
+  version: 9eb73efc1fcc404148b56765b0d3f61d9a5ef8ee
```

Comment:
❓ There's a lot going on between [these two commits](https://github.com/lib/pq/compare/50761b0867bd1d9d069276790bcd4a3bccf2324a...9eb73efc1fcc404148b56765b0d3f61d9a5ef8ee) -- is there anything in particular that you're after, or is this a chore-type update?

Reply:
This was necessary for `BeginTx` with transaction level support: https://github.com/lib/pq/commit/ed1a26d9e1fd4eb1cc491e6d688467f4e81a413c